release.go

```diff
@@ -19,13 +19,12 @@ func DefaultReleaseConfig() ReleaseConfig {
 }
 
 type Release struct {
-	bundles     []Bundle
-	changelogs  []Changelog
-	components  []Component
-	deprecated  bool
-	description string
```

Comment:
According to the api spec the description part is now dropped because we do not know how to aggregate it properly. I removed this within this PR since it is not that of a big deal.